html/canvas/offscreen/manual/the-offscreen-canvas/offscreencanvas.commit.html

```diff
@@ -14,20 +14,23 @@
     _assertPixel(canvas, 5,5, expectedR, expectedG, expectedB, expectedA, "5,5", expectedClrStr);
 }
 
-test(function() {
-    var placeholder = document.createElement('canvas');
-    placeholder.width = placeholder.height = 10;
-    var offscreenCanvas = placeholder.transferControlToOffscreen();
-    var ctx = offscreenCanvas.getContext('2d');
-    ctx.fillStyle = "#0f0";
-    ctx.fillRect(0, 0, 10, 10);
-    // commit() propagation is taken care of by an async task, which means the
-    // place holder contents should still be transparent black at this moment.
-    verifyPlaceholder(placeholder, 0,0,0,0, "0,0,0,0");
-    // Set timeout acts as a sync barrier to allow commit to propagate
-    setTimeout(function() {
-        verifyPlaceholder(placeholder, 0,255,0,255, "0,255,0,255");
-    }, 0);
+async_test(function(t) {
+    t.step(function() {
+        var placeholder = document.createElement('canvas');
+        placeholder.width = placeholder.height = 10;
+        var offscreenCanvas = placeholder.transferControlToOffscreen();
+        var ctx = offscreenCanvas.getContext('2d');
+        ctx.fillStyle = "#0f0";
+        ctx.fillRect(0, 0, 10, 10);
+        // commit() propagation is taken care of by an async task, which means the
+        // place holder contents should still be transparent black at this moment.
+        verifyPlaceholder(placeholder, 0,0,0,0, "0,0,0,0");
+        // Set timeout acts as a sync barrier to allow commit to propagate
+        setTimeout(t.step_func(function() {
```

Comment:
You can also use `t.step_timeout` here. There's a lint for use of `setTimeout` that would fail here, but it's been disabled for html/canvas/offscreen/manual/the-offscreen-canvas/*. Still, it's easily avoided here.